client/components/DisplayTest/index.jsx

```diff
@@ -120,19 +122,11 @@ class DisplayTest extends Component {
     }
 
     handleNextTestClick() {
-        if (this.testHasResult) {
-            this.performButtonAction('goToNextTest');
-        } else {
-            this.confirm('goToNextTest');
-        }
+        this.performButtonAction('goToNextTest');
     }
 
     handlePreviousTestClick() {
-        if (this.testHasResult) {
-            this.performButtonAction('goToPreviousTest');
-        } else {
-            this.confirm('goToPreviousTest');
-        }
+        this.performButtonAction('goToPreviousTest');
```

Comment:
In a follow up, "goToPreviousTest" should call triggger the "this.iframe.current.processResults"